test/e2e/pion/util.go

```diff
@@ -0,0 +1,58 @@
+package pion
+
+import (
+	"crypto"
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/hex"
+	"math/big"
+	"time"
+)
+
+func createSelfSigned() (*x509.Certificate, crypto.PrivateKey, error) {
```

Comment:
This is starting to pop up in multiple places now. Should we conciser adding it to the package?